cmd/main.go

```diff
@@ -29,6 +30,13 @@ func applianceMain() {
 		NewBuildCmd(),
 		NewCleanCmd(),
 		NewGenerateConfigCmd(),
+		NewGenerateUpgradeBundleCmd(),
```

Comment:
Can you please group all the upgrade related commands under some sub-command (i.e. rather than root).
Also, we need some documentation about the flow, as I'm a bit lost:)